src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.32bit.xml

```diff
@@ -1,6 +1,6 @@
 <linker>
   <assembly fullname="System.Private.CoreLib">
-    <type fullname="System.IntPtr">
+    <type fullname="nint">
```

Comment:
Is this valid? Looking through the other uses in this file, `int` is `System.Int32`, etc.

Reply:
No, its not. I thought I reverted this one. Will fix.